src/config-loader.js

```diff
@@ -1,14 +1,59 @@
-/** Config loader. */
-export default class ConfigLoader {
-  static async loadConfig() {
-    return fetch("/admin/config.json")
-      .then((response) => response.json())
-      .catch(() => {
-        // Defaults.
-        return {
-          api: "/api/",
-          touchButtonRegions: false,
-        };
-      });
-  }
-}
+// Only fetch new config if more than 5 minutes have passed.
+const configFetchInterval = 5 * 60 * 1000;
```

Comment:
This should not be hard coded. To options:
1. Add a value to https://github.com/os2display/display-admin-client/blob/develop/public/example_config.json
2. Load config once on user login. Config only reloaded on logout/login

Reply:
Will remove refresh of config